common/src/lib.rs

```diff
@@ -40,18 +40,13 @@ pub fn fast_hash(data: &[u8]) -> Hash {
 
 pub mod logger;
 
-// Logging related functionality
-cfg_if::cfg_if! {
-    if #[cfg(feature = "log")] {
-        mod panic_handler;
-
-        pub use panic_handler::setup_panic_handler;
-    }
-}
-
 // Loggers
 cfg_if::cfg_if! {
     if #[cfg(feature = "loggers")] {
+        mod panic_handler;
+
         pub mod sentry;
+
+        pub use panic_handler::setup_panic_handler;
```

Comment:
It only makes sense to provide the custom panic handler implementation together with the rest of the logging  infrastructure implementations (that are std-only and gated by the `loggers` feature).